values/json.go

```diff
@@ -0,0 +1,164 @@
+package values
+
+import (
+	"encoding/json"
+
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/semantic"
+)
+
+// JsonValue is a Flux value that can be encoded and decoded to/from JSON.
+type JsonValue struct {
+	Value
+}
```

Comment:
Why has this been added?

Reply:
This was for serializing the spec. I think it can be removed. I went into this with the intention to preserve the repl.